utils.sh

```diff
@@ -584,6 +584,28 @@ function wait_for_crd() {
   oc wait --for condition=established --timeout=60s "crd/$1" || exit 1
 }
 
+function generate_proxy_conf() {
+  echo "acl all src $PROVISIONING_NETWORK"
+
+  if [[ ! -z "${EXTERNAL_SUBNET_V4}" ]]
+  then
+    echo "acl all src $EXTERNAL_SUBNET_V4"
+  fi
+
+  if [[ ! -z "${EXTERNAL_SUBNET_V6}" ]]
+  then
+    echo "acl all src $EXTERNAL_SUBNET_V6"
+  fi
+
+  cat <<EOF
```

Comment:
If you move the `EXT_SUBNET` calculation into this (or some other common) file, the conditionals could be replaced with e.g:

```
acl all src $PROVISIONING_NETWORK
acl all src $EXT_SUBNET
http_access allow all
http_port ${INSTALLER_PROXY_PORT}
debug_options ALL,2
dns_v4_first on
coredump_dir /var/spool/squid
EOF
```

Although having said that, do we need to omit the PROVISIONING_NETWORK when `PROVISIONING_NETWORK_PROFILE` is `Disabled` ?